docs/docs/sample-applications/3-streaming-ffmpeg.md

```diff
@@ -1,15 +1,15 @@
 # A Quick Example: Live Stream Alert
```

Comment:
Should this be removed or replaced by your new way of processing streams described in Low-Code Stream Processor?